admin/admin-ui-render-settings.php

```diff
@@ -297,6 +298,40 @@ function superpwa_orientation_cb() {
 	<?php
 }
 
+/**
+ * Default Display Dropdown
+ *
+ * @since 2.0
+ */
```

Comment:
Oh, forgot to mention this. Please keep the @author before @since. @since will keep changing as we change the functions, so its nicer to be the last item. Like these: https://github.com/SuperPWA/Super-Progressive-Web-Apps/blob/a06f52e1b0ddb74057ffb6d22545a9ae285ffe12/public/manifest.php#L109-L111

Reply:
@josevarghese Can you please add the @author and lets merge this?